src/Form/Form.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import './Form.scss';
+
+export default function Form(props) {
+  return (
+    <form
+      className={classNames('Form', props.inline ? 'inline' : '')}
```

Comment:
Should this be:
```suggestion
      className={classNames('Form', props.inline ? 'Form--inline' : '')}
```
Also, using classNames this can be:
```suggestion
      className={classNames('Form', { 'Form--inline': props.inline })}
```